### ISSUE TITLE
Can I get ViewerLocalFileData to open a fully-fledged instance of the OHIF Viewer?

### ISSUE THREAD

**Issue by omartoutounji** [OHIF/Viewers#833]
I am trying to allow users to drag-and-drop their DICOM data to OHIF. I noticed that isn't supported with the main interface of the OHIF Viewer unless you add a `/local` to the website.

I am wondering if there's any way I can get the local viewer (`ViewerLocalFileData.js` or `https://viewer.ohif.org/local`) to open an instance of the OHIF Viewer with all of its tools/capabilities instead of a half-baked version as it does right now.

Does the current OHIF architecture support this use case? If not, can I get guidance to draft a PR. It would be ideal if I could ignore the local viewer altogether and just add the drag and drop feature to OHIF and see my study there instead of studies from the server.

TL;DR Trying to get drag and drop working with OHIF

**Comment by swederik** [comment]
Some bugs I know of:
- DICOM Segmentation files aren't loaded correctly, since the extension currently uses dicomweb-client to try to retrieve the file over WADO-URI
- Same deal with DICOM Slide Microscopy images
- Possibly same deal with structured reports

Otherwise I think it should be basically the same viewer, but I haven't done a ton of testing. I'd be happy to have these problems resolved. They are very similar to the same problems we are having with the Google Cloud integration.

Anything else you have noticed that is broken?

**Comment by omartoutounji** [comment]
> Some bugs I know of:
> 
> * DICOM Segmentation files aren't loaded correctly, since the extension currently uses dicomweb-client to try to retrieve the file over WADO-URI
> * Same deal with DICOM Slide Microscopy images
> * Possibly same deal with structured reports
> 
> Otherwise I think it should be basically the same viewer, but I haven't done a ton of testing. I'd be happy to have these problems resolved. They are very similar to the same problems we are having with the Google Cloud integration.
> 
> Anything else you have noticed that is broken?

Yes, none of the cornerstone tools are there

**Comment by swederik** [comment]
That's weird, they seem to be working for me...

![Uploading Screenshot 2019-08-30 18.34.25.png…]()

However it appears the measurement panel is broken (even for non-local usage), and same with the right and middle click tools. I think these are both side effects of the change to the monorepo.

**Comment by omartoutounji** [comment]
> That's weird, they seem to be working for me...
> 
> ![Screenshot 2019-08-30 18 34 25](https://user-images.githubusercontent.com/607793/64037076-04851500-cb55-11e9-8800-eaaa9788b853.png)
> 
> However it appears the measurement panel is broken (even for non-local usage), and same with the right and middle click tools. I think these are both side effects of the change to the monorepo.

Interesting, I used different data and the tools started showing. Maybe invalid DICOM files disable the tools?

**Comment by dannyrb** [comment]
Scoped in #838

**Comment by Srikanthkurva** [comment]
Hi I have a question I'm successfully running this application. Viewers. But i want I want to replace with DCOM Images. How to replace
